Pipfile

```diff
@@ -0,0 +1,76 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+boto3 = "~=1.12.1"
+botocore= "~=1.15.1"
+flake8= "~=3.7.8"
+pep8= "~=1.7.1"
+pyflakes= "~=2.1.1"
+pycodestyle= "~=2.5.0"
+autopep8= "~=1.4.4"
+Babel= "~=2.7.0"
+beautifulsoup4= "~=4.8.0"
+cachetools= "~=3.1.1"
+coverage= "~=4.5.4"
+decorator= "~=4.4.0"
+docutils= "~=0.15.2"
+esrijson= "~=0.4.3"
+future= "~=0.17.1"
+gatilegrid= "~=0.1.19"
+GeoAlchemy2="~=0.6.3"
+geojson= "~=2.5.0"
+hupper= "~=1.8.1"
+Jinja2= "~=2.11.3"
+lark_parser= "~=0.7.7"
+lxml= "~=4.6.3"
+Mako= "~=1.1.0"
+MarkupSafe= "~=1.1.1"
+mccabe= "~=0.6.1"
+# TODO : when upgrading to python 3.9, we can have networkx use a ~= to get the latest minor version
+# TODO : but it's currently not python 3.7 compatible.
+networkx= "==2.2"
+nose= "~=1.3.7"
+papyrus= "~=2.4"
+pip= "~=21.2.4"
+numpy= "~=1.16.5"
+PasteDeploy= "~=2.0.1"
+Pillow= "~=8.3.2"
+polib= "~=1.1.0"
+psycopg2-binary= "~=2.8.3"
+Pygments= "~=2.7.4"
+pyproj= "~=1.9.6"
+pyramid= "~=1.10.4"
+pyramid-debugtoolbar= "~=4.5"
+pyramid-mako= "~=1.1.0"
+pyramid-tm= "~=2.3"
+PyRSS2Gen= "~=1.1"
+python-dateutil= "~=2.8.0"
+pytz= "~=2019.2"
+PyYAML= "~=5.4.1"
+regex= "~=2019.8.19"
+"repoze.lru"= "~=0.7"
+requests= "~=2.22.0"
+Shapely= "~=1.6.4.post2"
+simplejson= "~=3.16.0"
+# TODO: clean-up when only Python 3.x and no longer 2.x is in use
+six= "~=1.16.0"
+Sphinx= "~=1.8.5"
+sphinx-rtd-theme= "~=0.4.3"
+SQLAlchemy= "~=1.3.24"
+toml= "~=0.10.0"
+transaction= "~=2.4.0"
+translationstring= "~=1.3"
+unittest-xml-reporting= "~=2.5.2"
+venusian= "~=1.2.0"
+waitress= "~=1.4.3"
+WebOb= "~=1.8.5"
+WebTest= "~=2.0.33"
+"zope.deprecation"= "~=4.4.0"
+"zope.interface"= "~=4.6.0"
+unidecode= "~=1.1.1"
+
```

Comment:
You need to add `[dev-packages]` section even if empty in order to pipenv to work correctly with `install -e .`

Reply:
Using here `~=` resolve to some dependencies warning, might be a good idea to stick to `==` like in pip for this PR.